addon/components/search/result-table.hbs

```diff
@@ -1,8 +1,11 @@
-<table
-  class="uk-table uk-table-middle uk-table-divider uk-table-striped"
-  ...attributes
->
-  {{yield
-    (component "search/result-table/row" results=@results loading=@loading)
-  }}
-</table>
+{{#if (gt @results.length 0)}}
```

Comment:
Nitpick: length is falsy if 0 so `{{#if @result.length}}` would be enough .
```suggestion
{{#if (gt @results.length 0)}}
```

Reply:
has been removed